### PR TITLE
`default-branch-button` - Fix pink button on GHE

### LINKED ISSUE #7857: `default-branch-button`: Button has pink corruption on GHES
### Description

On Github Enterprise Server, the `default-branch-button` feature causes the branch button to appear with pink corruption. 

image

### How to replicate the issue + URL

1. Have access to a GHES server (3.12.7)
2. Ensure a PAT for the server is added to the extension
3. Navigate to the equivalent of https://github.com/refined-github/refined-github/tree/sandbox/keep-branch
4. Observe ugly pink corruption.

### Extension version

24.10.1

### Browser(s) used

Firefox (130.0 Win64)

### Token ⚠️

- [X] I set a token in the options page. I understand that some features require a token to work.

### PR DESCRIPTION
**Pull request by JakeMerdichAMD**
Adding old css variables and tweaking the selectors so this doesn't render incorrectly on older GHES platforms.

Fixes #7857

## Test URLs
https://github.com/refined-github/refined-github/tree/sandbox/keep-branch?rgh-link-date=2024-10-04T15%3A40%3A11Z

## Screenshot
https://github.com/refined-github/refined-github/issues/7857#issuecomment-2394150829

### CODE REVIEW

**Comment by fregante** [comment]
You can run `npm run fix` to fix everything at once

**Comment by JakeMerdichAMD** [comment]
Thank you! I'm not super familiar with javascript tooling and was struggling to find what to fix.

**Comment by fregante** [comment]
For this to be merged though you have to make sure that it doesn’t break the dropdown, otherwise we’re back to square one. Your “screenshot” link points to those broken screenshots.

**Comment by JakeMerdichAMD** [comment]
This changes fixes the dropdown, not breaks it. I included that image for 'before' because it wasn't clear what was being fixed otherwise.

**Comment by fregante** [comment]
Ahhh I had misread your message because you put the “before” picture *after* the fixed one. 🙂

File: source/features/default-branch-button.css [hunk 1]
```diff
@@ -1,9 +1,17 @@
-.rgh-highlight-non-default-branch {
-	background-color: var(--bgColor-accent-muted, fuchsia) !important;
-	color: var(--fgColor-accent, fuchsia) !important;
-	border-color: var(--fgColor-accent, fuchsia) !important;
+button.rgh-highlight-non-default-branch,
```
Comment (fregante) [inline source/features/default-branch-button.css:1]:
```suggestion
/* TODO: Revert https://github.com/refined-github/refined-github/pull/7859 in March 2025 */
/* The `details` version is for GHE. The `button` in the selector avoids a GHE conflict */
button.rgh-highlight-non-default-branch,
```

**Review by fregante — APPROVED** [review]

**Comment by fregante** [comment]
I hotfixed this CSS via https://github.com/refined-github/yolo/commit/59963196adcaeb1563c186d3a60be7315d8d7ac9 in v24.10.1

**Pull request merged**